docs/docs/tutorials/ssh-signing.md

```diff
@@ -0,0 +1,103 @@
+# :material-key: SSH and Commit Signing
+
+This tutorial will guide you through setting up SSH keys and commit signing for GitHub. SSH keys are a more secure way to connect to GitHub, rather than other protocols like HTTPS. Commit signing is used to verify that commits are coming from you and have not been tampered with.
+
+!!! warning
+    You must have a verified email address on GitHub to add SSH keys and sign commits.
+
+## Create an SSH Key
+
+=== "Windows"
+
+    In an admin PowerShell terminal, enter the following command:
+
+    ```text
+    ssh-keygen -t ed25519 -C "your_github_email@example.com"
+    ```
+
+    Save to the default location `c:/Users/YOU/.ssh/id_ed25519` and do not enter a passphrase. This will allow you to use the key without entering a password every time you push to GitHub.
+
+    Continue with the following commands:
+
+    ```text
```

Comment:
When I went through this tutorial, I didn't need to run block of code (I was able to ssh into GitHub without running it). I think these commands come from when you had to install openssh from the command line. now there is a package which handles everything. My commit adds instructions for that.

Reply:
Realized this can be done in a `bash` terminal for all platforms. Updated docs accordingly. When you SSH'd into GitHub were you connecting to the SSH remote origin or HTTPS? See later comment for more details.

Reply:
Yes, that makes sense since the key is configured globally. The command only required the authentication key to be added to your GitHub account.

Weird how that worked without the extra `eval` and `ssh-add` commands. We usually need to manually add the key to the agent and known hosts using them. Either you already had SSH set up or OpenSSH automatically handled the whole thing.

New docs don't required any software like OpenSSH to be installed since everything works in bash.